src/core/nixl_tcpstore_metadata_backend.cpp

```
@@ -0,0 +1,189 @@
+
+    const long timeout_ms =
+        nixl::config::getValueDefaulted<long>("NIXL_TCPSTORE_TIMEOUT_MS", kDefaultTimeoutMs);
+
+    client_ = std::make_unique<nixlTcpStoreClient>(
+        host, static_cast<std::uint16_t>(port), std::chrono::milliseconds(timeout_ms));
+    NIXL_DEBUG << "[" << ctx_.getName() << "] connected TCPStore client to " << endpoints;
+}
+
+nixlTcpStoreMetadataBackend::~nixlTcpStoreMetadataBackend() = default;
+
+nixl_status_t
```

Cppcheck (2.21.0)

[style] The function 'unlock' is never used.

(unusedFunction)